internal/validator/validator.go

```diff
@@ -142,6 +163,11 @@ func ValidateMessage(message *parser.Message) (string, error) {
 		return "", fmt.Errorf("%s", err)
 	}
 
+	// Validate the commit message length
+	if err := checkMessageLength(&message.Description); err != nil {
```

Comment:
You are supposed to check the entire "commit message" and not just the description of the message.

A commit message is structured as - `<TYPE>[OPTIONAL SCOPE]: <DESCRIPTION>` ([see the summary of the Conventional Commits specifications](https://www.conventionalcommits.org/en/v1.0.0/#summary)). An example commit message is - `"chore(parser): improve parsing performance of the message"`.